tests/test_float_generator.py

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
When we setup pre-commit, we can clean this then if no other changes are necessary.